bvm/ballerina-profiler/src/main/java/io/ballerina/runtime/profiler/ui/FrontEnd.java

```diff
@@ -0,0 +1,331 @@
+/*
+ * Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com) All Rights Reserved.
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.ballerina.runtime.profiler.ui;
+
+/**
+ * This class contains the front end of the Ballerina profiler.
+ *
+ * @since 2201.8.0
+ */
+public class FrontEnd {
+
+    String getSiteData(String contents) {
+        return "<!DOCTYPE html>\n" +
+                "<html lang=\"en\">\n" +
+                "\n" +
+                "<head>\n" +
+                "   <meta charset=\"utf-8\">\n" +
+                "   <meta http-equiv=\"X-UA-Compatible\" content=\"IE=edge\">\n" +
+                "   <meta name=\"viewport\" content=\"width=device-width, initial-scale=1\">\n" +
```

Comment:
In a separate PR we can move this to an html file and process that.

Reply:
There is an issue with passing the file directory. I will address this in https://github.com/ballerina-platform/ballerina-lang/issues/41224